runner.go

```diff
@@ -530,6 +530,32 @@ func getPlatforms(c cliContext) []string {
 	return expandStringSlice(platforms)
 }
 
+func getExecutionMethod(c cliContext) (string, error) {
+	var crowd string
+	if crowd = c.String("crowd"); crowd != "" {
+		fmt.Println("RF CLI Deprecation: --crowd is deprecated, use --execution-method instead")
+	}
+	executionMethod := c.String("execution-method")
+
+	if crowd != "" && executionMethod != "" {
+		return "", errors.New("execution-method and crowd are mutually exclusive")
```

Comment:
Wait, what? I thought 'crowd' is just deprecated?

Reply:
`--execution-method` replaces `crowd`. The two parameters are already mutually exclusive at the API level, this just bubbles that existing validation up to the CLI.